WRR-4833: Make pageIndicatorPosition prop in PageViews component public (#1757)
enactjs/sandstone: CHANGELOG.md

### Added

- `sandstone/Icon` supported icon list, adding new icon `calibration`

> - `sandstone/Pageviews` `pageIndicatorPosition` prop to provide a way to determine where to place the page indicator

Reviewer: ```suggestion
- `sandstone/Pageviews` `pageIndicatorPosition` prop to set the position of page indicator
```
